nixos/modules/services/web-servers/meguca.nix

```diff
@@ -86,11 +86,19 @@ in with lib; {
       default = false;
       description = "Serve and listen only through HTTPS.";
     };
+
+    videoPaths = mkOption {
+      type = types.listOf types.path;
+      default = [];
+      example = [ "/home/okina/Videos/tehe_pero.webm" ];
+      description = "Videos that will be symlinked into www/videos.";
+    };
   };
 
   config = mkIf cfg.enable {
     security.sudo.enable = cfg.enable;
     services.postgresql.enable = cfg.enable;
+    services.postgresql.package = pkgs.postgresql_11;
```

Comment:
Why?
github pages says:
> Runtime dependencies
> PostgresSQL >= 10.0

Reply:
When given the choice, use the higher version, or at least that's how I see it, since I'm already having to specify.

Reply:
In that case we should have updated default postgresql version to 11 everywhere in NixOS. I don't think it's a good idea to dump parts of personal configuration into common NixOS modules.

Reply:
I don't think that was my intention, but in that case, wouldn't making the default version 10 be effectively the same thing?
The problem is, if it isn't at least version 10, then meguca won't function.

Reply:
Okay. Sorry, my bad, I did not realize we are using 9.6 by default:
https://github.com/NixOS/nixpkgs/blob/6f95ac3588597a2d165f09b77dc2e607c1f286b9/nixos/modules/services/databases/postgresql.nix#L170-L172
This is fine then.

Reply:
No problem. :P